Gems/PhysX/Code/Tests/PhysXTestCommon.cpp

```diff
@@ -63,6 +64,16 @@ namespace PhysX
             }
         }
 
+        float CalculateZFromFalling(float gravityMultipler, float totalTimeSteps, float startingZPosition)
+        {
+            // Calculate distance fallen (d = 0.5 * g * t^2)
+            float gravity = startingZPosition * gravityMultipler;
+            float time = totalTimeSteps * AzPhysics::SystemConfiguration::DefaultFixedTimestep;
+
+            return 0.5f * (gravity) * powf(time, 2);
```

Comment:
uber nit: Change `powf(time, 2)` to just `time * time` (it's also good to prefer `AZStd::pow` if you do need to use it

Reply:
Noted in internal backlog ticket and removing from this PR since this code is not used for the remaining test for the focus of this PR.

GHIs will be created this week.